packages/udes-ui-validation/src/exampleComponent/materialUiComponent.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { useState } from "react";
```

Comment:
Please use Storybook.js for examples

Reply:
I'm a bit confused by this comment, this demonstrates how to implement the code? As a package this should be IMHO be separate from storybook

Reply:
I assumed this was to show-off usage.  npm packages usually have coded examples in the package README.md so it's published to the packages' npmjs.com page (developers don't look in their node_modules folder typically as it's huge).